src/app/collective-rewards/components/CallToActionSection/CallToActionSection.tsx

```diff
@@ -0,0 +1,51 @@
+import { useGetCycleRewards } from '../../shared/hooks/useGetCycleRewards'
+import { useGetEstimatedRewardsPct } from '../../shared'
+import { BackersCallToAction } from '../BackersCallToAction'
+import { BuildersCallToAction } from '../BuildersCallToAction'
+import { InfoContainer } from '@/components/containers'
+import { WeiPerEther } from '@/lib/constants'
+import { LoadingSpinner } from '@/components/LoadingSpinner'
+import { useHandleErrors } from '../../utils'
+
+export const CallToActionSection = () => {
+  const { data: builders, isLoading: buildersLoading, error: buildersError } = useGetEstimatedRewardsPct()
+  const {
+    data: cycleRewards,
+    isLoading: cycleRewardsLoading,
+    error: cycleRewardsError,
+  } = useGetCycleRewards()
+
+  const isLoading = buildersLoading || cycleRewardsLoading
+  const error = buildersError ?? cycleRewardsError
+  useHandleErrors({ error, title: 'Error loading CTA section' })
+
+  const totalRewardsPct = builders.reduce(
+    (acc, builder) => {
+      const { estimatedBackerRewardsPct, estimatedBuilderRewardsPct } = builder
+      return {
+        estimatedBackerRewardsPct: acc.estimatedBackerRewardsPct + estimatedBackerRewardsPct,
+        estimatedBuilderRewardsPct: acc.estimatedBuilderRewardsPct + estimatedBuilderRewardsPct,
+      }
+    },
+    { estimatedBackerRewardsPct: 0n, estimatedBuilderRewardsPct: 0n },
+  )
+
+  const rifBuilderRewards =
+    (totalRewardsPct.estimatedBuilderRewardsPct * cycleRewards.rifRewards) / WeiPerEther
+  const rbtcBuilderRewards =
+    (totalRewardsPct.estimatedBuilderRewardsPct * cycleRewards.rbtcRewards) / WeiPerEther
+  const rifBackerRewards = (totalRewardsPct.estimatedBackerRewardsPct * cycleRewards.rifRewards) / WeiPerEther
+  const rbtcBackerRewards =
+    (totalRewardsPct.estimatedBackerRewardsPct * cycleRewards.rbtcRewards) / WeiPerEther
```

Comment:
Would it make more sense to load this data directly inside the backers/Builder CalltoAction components?